submissions/ArthurGorbenko/frogger-game/js/app.js

```diff
@@ -0,0 +1,130 @@
+let countForWonRound = 0;
+
+const blockWidth = 100;
+const blockHeight = 90;
+const fieldMaxHeight = 400;
+const fieldMinHeight = -70;
+const fieldMaxWidth = 405;
+const fieldMinWidth = -10;
+const waterLevel = fieldMinHeight + blockHeight;
+const startPositionX = 200;
+const startPositionY = 300;
+
+class Enemy {
+  constructor(x, y) {
+    this.sprite = "images/enemy-bug.png";
+    this.x = x;
+    this.y = y;
+    this.speed = Math.floor(Math.random() * 100) + 50;
+  }
+  update(dt) {
+    this.x += this.speed * dt;
+
+    if (this.x > 600) {
```

Comment:
Please avoid "magic numbers" in your code. Use named constants instead. This makes your code easier to read and also to maintain. Imagine that one day you will need to change 600 to 700: you would have to go through your code and make sure you did not miss any occurrence of this value.

Reply:
Thak's for your detailed response. That is usefull! ;)